Add note about firefox extension.

JJK96 opened the pull request:
Ref #1

JJK96 committed:
Add note about firefox extension.
Ref #1

alexherbo2 commented:
Can you add the [copyright waiver][CONTRIBUTING] before the first commit?

[CONTRIBUTING]: https://github.com/alexherbo2/krabby/blob/master/CONTRIBUTING

JJK96 committed:
Jan-Jaap Korpershoek Copyright Waiver

I dedicate any and all copyright interest in this software to the
public domain.  I make this dedication for the benefit of the public at
large and to the detriment of my heirs and successors.  I intend this
dedication to be an overt act of relinquishment in perpetuity of all
present and future rights to this software under copyright law.

JJK96 commented:
done

alexherbo2 commented:
Thanks!
